includes/utils/class-sites.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Newspack Network Sites methods.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network\Utils;
+
+use Newspack_Network\Site_Role;
+use Newspack_Network\Hub\Nodes;
+use Newspack_Network\Hub\Node as Hub_Node;
+use Newspack_Network\Node\Settings as Node_Settings;
+
+/**
+ * Class to get information about the sites in the network.
+ *
+ * Helper methods to get sites lists and info in an easy way.
+ *
+ * This is specially useful for third party integration that wants to get a list of the sites in the network.
+ * With these methods, you can get that lists without worrying if you are in a hub or a node and how the data is stored.
+ */
+class Sites {
+
+	/**
+	 * Get all sites
+	 *
+	 * @return array Array of arrays with name and url properties.
+	 */
+	public static function get_all_sites() {
+		return array_merge( [ self::get_hub() ], self::get_nodes() );
+	}
+
+	/**
+	 * Get all sites but the current one
+	 *
+	 * @return array Array of arrays with name and url properties.
+	 */
+	public static function get_all_sites_but_current() {
```

Comment:
Nit: 

```suggestion
	public static function get_all_sites_without_current() {
```

IMO that's more readable

Reply:
no strong preference: applied in https://github.com/Automattic/newspack-network/pull/262/commits/a742a4492431ecf6201a5f7dd34b6fb0a7f9e8bd